### ISSUE TITLE
inspect() not in TS definitions

### ISSUE THREAD

**Issue by mattbishop** [eddow/ts-json-schema-decorator#2]
I have no way of generating a JSON Schema object (or even string) without this function!

**Comment by eddow** [comment]
Can you rephrase please? the schema is generated into a propriety of the class - even just `schema` I think

**Comment by eddow** [comment]
Are you speaking of `util.inspect` ?

**Comment by taxilian** [comment]
@mattbishop it looks to me like inspect in the example is just showing you that there is a static member `schema` on the object. There are a few ways you could expose that or get to it.
